plugins/airbrake-backend/src/config/ExtractAirbrakeConfig.ts

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright 2022 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { Config } from '@backstage/config';
+
+/**
+ * The Airbrake config object
+ *
+ * @public
+ */
+export type AirbrakeConfig = {
+  /**
+   * The API Key
+   */
+  apiKey: string;
+};
+
+/**
+ * Extract the Airbrake config from a config object
+ *
+ * @public
+ *
+ * @param config - The config object to extract from
+ */
+export function extractAirbrakeConfig(config: Config): AirbrakeConfig {
+  return {
+    apiKey: config.getString('airbrake.apiKey'),
```

Comment:
Probably makes sense to add a config schema to the backend plugin so that we can be sure that this is going to be here at startup otherwise this will crash. Here's a guide: https://backstage.io/docs/conf/defining

Reply:
Thanks for the review, I've done this I think, any chance you could see if I did it correctly @benjdlambert ?